src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrayWithCopies = [];
+  const copy = { ...state };
+
+  for (const object of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
avoid meaningless naming like object for variables